apis/python/CMakeLists.txt

```diff
@@ -0,0 +1,128 @@
+#
+# The MIT License
+#
+# Copyright (c) 2024 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+cmake_minimum_required(VERSION 3.21)
+
+project(
+    ${SKBUILD_PROJECT_NAME}
+    VERSION ${SKBUILD_PROJECT_VERSION}
+    LANGUAGES CXX
+)
+
+find_package(pybind11 REQUIRED)
+
+# Search for libtilebdvcf in the default install directory and in the directory
+# specified by the LIBTILEDBVCF_DIR environment variable.
+set(LIBTILEDBVCF_SEARCH_PATHS 
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../dist/lib" 
+    "$ENV{LIBTILEDBVCF_DIR}"
+)
+
+message(STATUS "Searching for libtiledbvcf in ${LIBTILEDBVCF_SEARCH_PATHS}")
+
+# TODO: Use find_package() instead of find_library() to find libtiledbvcf.
+find_library(
```

Comment:
Can't we use find_package? Or if this is not a different package but build by this project then add_subdirectory?

Reply:
Yes, that was @ihnorton's comment too. The `libtiledbvcf` cmake files will need some work to support a different approach. I'll add that task as a backlog item instead of blocking this PR.